test/integration/targets/iosxr_l3_interfaces/tests/cli/merged.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- debug:
+      msg: "START Merged iosxr_l3_interfaces state for integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- block:
+    - name: Merge provided configuration with device configuration
+      iosxr_l3_interfaces: &merged
+        config:
+          - name: GigabitEthernet0/0/0/2
+            ipv4:
+            - address: 192.168.0.2/24
```

Comment:
Please use documentation IP addresses in tests - https://tools.ietf.org/html/rfc5737